### PR TITLE
Fix erroneous ends

### PR DESCRIPTION
**Pull request by westonio**
# Describe the changes below:
- Fixes erroneous end statements
- Removes unused lines of code
- Organizes merchant spec files

# Before submitting, check the following:
- [x] Entire test suite is passing
- [99.22%] SimpleCov test percentage

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -44,10 +44,6 @@ header {
   color: #4A4A48;
 }
 
-#top-merchants {
-  
-}
```
Comment (awiley33) [inline app/assets/stylesheets/application.css:49]: nice catch removing that!

**Review by MatthewTLim — APPROVED** [review]
Nice job removing the unnecessary debugs, this will help with our test coverage.

**Review by dani-wilson — APPROVED** [review]
Thanks for doing this cleanup!

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -31,11 +27,9 @@ def get_random_photo
 
   def search_item_photos(search_term, page: 1, per_page: 1, order_by: 'relevant')
```
Comment (awiley33) [inline app/controllers/application_controller.rb:28]: way to remove these lines and increase our simplecov

File: app/models/merchant.rb [hunk 3]
```diff
@@ -33,7 +33,7 @@ def best_merchant_sales_day
     .group('date').order('total_revenue DESC')
     .limit(1).first.date
   end
-end
+
```
Comment (awiley33) [inline app/models/merchant.rb:36]: nice job finding this

**Review by awiley33 — APPROVED** [review]

**Pull request merged**